x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/index.tsx

```diff
@@ -237,6 +238,16 @@ const StepAboutRuleComponent: FC<StepAboutRuleProps> = ({
               }}
             />
             <EuiSpacer size="l" />
+            <UseField
+              path="customHighlightedFields"
+              component={MultiSelectFieldsAutocomplete}
+              componentProps={{
+                browserFields: indexPattern.fields,
+                isDisabled: isLoading || indexPatternLoading,
+                fullWidth: true,
+              }}
+            />
```

Comment:
So we do auto-suggest values, just based on the source events mappings rather than the alerts index mappings? Maybe I missed this part over zoom. The reason I'm asking is that with this auto-suggest component, it's not possible to add a field that is not in the source mappings. Is it by design?